src/class-wpml-gutenberg-strings-in-block.php

```diff
@@ -58,7 +58,7 @@ public function find( WP_Block_Parser_Block $block ) {
 
 		}
 
-		return $strings;
+		return apply_filters( 'wpml_block_parser_found_strings', $strings, $block );
```

Comment:
To be more consistent, I would rename this `wpml_found_strings_in_block`.

Reply:
done